msal4j-sdk/src/main/java/com/microsoft/aad/msal4j/ManagedIdentityParameters.java

```diff
@@ -15,7 +15,7 @@ public class ManagedIdentityParameters implements IAcquireTokenParameters {
     String resource;
     boolean forceRefresh;
     String claims;
-
+    String revokedTokenHash;
     private ManagedIdentityParameters(String resource, boolean forceRefresh, String claims) {
```

Comment:
nit: very minor, just adding a blank line between all the fields and the first method

```suggestion
    String revokedTokenHash;
    
    private ManagedIdentityParameters(String resource, boolean forceRefresh, String claims) {